crates/subspace-farmer/src/farmer_cache.rs

```diff
@@ -176,38 +190,38 @@ where
             // TODO: Consider implementing optional re-sync of the piece instead of just forgetting
             WorkerCommand::ForgetKey { key } => {
                 let mut caches = self.piece_caches.write().await;
+                let Some(offset) = caches.stored_pieces.remove(&key) else {
+                    // key not exist.
```

Comment:
NIT: please use this formatting going forward:
```suggestion
                    // Key not exist
```

To be consistent with existing comments